src/routes/docs/products/storage/quick-start/+page.markdoc

```diff
@@ -119,7 +119,28 @@ To upload a file, add this to you web, Flutter, Apple, or Android app.
 
   --cec8e8123c05ba25--
   ```
+  ```client-react-native
+  import { Client, Storage, ID } from "react-native-appwrite";
 
+  const client = new Client()
+    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('<PROJECT_ID>');                 // Your project ID
+
+  const storage = new Storage(client);
+
+  const promise = await storage.createFile(
+    '[BUCKET_ID]',
+    ID.unique(),
+    {name: "file.txt", type: "text/plain", size: 37967, uri: ""}, // FILE
```

Comment:
What kind of tests have you done to make sure this works? The blank `uri` is pretty confusing here.